documentation/book/proc-scaling-down-a-kafka-cluster.adoc

```diff
@@ -22,10 +22,19 @@ You can verify this by checking that the broker's data log directory does not co
 If the log directory on the broker contains a directory that does not match the extended regular expression `[a-zA-Z0-9.-]+\.[a-z0-9]+-delete$` then the broker still has live partitions and it should not be stopped.
 +
 You can check this by executing the command:
+ifdef::Kubernetes[]
```

Comment:
Are you sure `oc` cannot do `oc exec ...`?

Reply:
`oc exec` exists, of course. There are numerous places where we document an `oc` command where it is simpler/"better" than the equivalent `kubectl`. I'm happy to change the policy here and always use `{cmdcli}` and only document `oc` separately if there is not corresponding command in `kubectl` (e.g. s2i stuff). @PaulRMellor @laidan6000 I assume you're happy with this?